airflow/utils/helpers.py

```diff
@@ -30,6 +30,8 @@
 from airflow.configuration import conf
 from airflow.exceptions import AirflowException
 
+BACKFILL_PREFIX = "backfill_"
```

Comment:
This feels a bit out of place here in "helpers". This isn't a helper, it's a constant to do with jobs.

I think this would be better placed in either base_jobs.py, or `jobs/__init__.py`. That or this variable should become `BACKFILL_JOB_PREFIX.` or `JOB_PREFIX_BACKFILL`

Reply:
I agree, but I was unable to find better place (and wanted to avoid __init__). Other option I considered was to create new `airflow/utils/constants.py` WDYT? @ashb

Reply:
How about airflow/utils/backfill.py and renaming to BACKFILL_JOB_PREFIX ?

Reply:
Let's do this in a way that will be extensible: `airflow/utils/jobs.py`

Reply:
I'd marginally prefer `airflow/jobs/constants.py` - `utils` is a bit of a "dumping ground" - we may as well call it `airflow.collectionofjunk` :)

Reply:
This should not include the word "Job" but DagRun.  We have type "manual_" which is not associated with any job.